tests/ui/legacy_numeric_constants.stderr

```diff
@@ -0,0 +1,106 @@
+error: usage of a legacy numeric constant
+  --> $DIR/legacy_numeric_constants.rs:30:5
+   |
+LL |     std::f32::EPSILON;
+   |     ^^^^^^^^^^^^^^^^^ help: use the associated constant instead: `f32::EPSILON`
```

Comment:
This is just a suggestion to try out: Having long inline help messages makes it harder for me to understand what they want at a glance. Maybe, this would look clear and cleaner when you use [span_suggestion_with_style](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/diagnostic/struct.Diag.html#method.span_suggestion_with_style) with [SuggestionStyle::ShowAlways](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/enum.SuggestionStyle.html#variant.ShowAlways)

The result should look roughly like this

```
 --> $DIR/legacy_numeric_constants.rs:30:5
   |
LL |     std::f32::EPSILON;
   |     ^^^^^^^^^^^^^^^^^
   = help use the associated constant instead:
   |
LL |     f32::EPSILON;
   |
```

Reply:
Wasn't aware of the different styles, thanks for the tip.